[ci-visibility] Update cypress instructions to warn about `cypress open` (#17298)
DataDog/documentation: content/en/continuous_integration/tests/javascript.md

If you want visibility into the browser process, consider using [RUM & Session Replay][10]. When using Cypress, test results and their generated RUM browser sessions and session replays are automatically linked. Learn more in the [RUM integration][11] guide.

### Cypress interactive mode

> Cypress interactive mode (cypress is run through `cypress open`) is not supported by CI Visibility, because some cypress events are not fired, such as [`before:run`][12]. If you want to try it anyway, pass `experimentalInteractiveRunEvents: true` to the [cypress configuration file][13].

Reviewer: ```suggestion
Cypress interactive mode (which you can enter by running `cypress open`) is not supported by CI Visibility because some cypress events, such as [`before:run`][12], are not fired. If you want to try it anyway, pass `experimentalInteractiveRunEvents: true` to the [cypress configuration file][13].
```
